contracts/DaiHrd.sol

```diff
@@ -486,38 +479,38 @@ contract DaiHrd is ERC777, MakerFunctions {
 		}
 	}
 
-	function deposit(uint256 attodai) external returns(uint256 depositedAttopot) {
+	function deposit(uint256 attodai) external returns(uint256 attodaiHrd) {
 		dai.transferFrom(msg.sender, address(this), attodai);
 		daiJoin.join(address(this), dai.balanceOf(address(this)));
-		depositedAttopot = depositVatDaiForAccount(msg.sender);
+		uint256 depositedAttopot = depositVatDaiForAccount(msg.sender);
 		emit Deposit(msg.sender, attodai, depositedAttopot);
 		return depositedAttopot;
 	}
 
-	// If the user has vat dai directly (after performing vault actions, for instance), they don't need to create the DAI ERC20 for us to burn it, we'll accept vat dai
-	function depositVatDai(uint256 attoVatDai) external returns(uint256 depositedAttopot) {
-		uint256 attorontodai = attoVatDai.mul(10 ** 27);
-		vat.move(msg.sender, address(this), attorontodai);
-		depositedAttopot = depositVatDaiForAccount(msg.sender);
-		emit DepositVatDai(msg.sender, attorontodai, depositedAttopot);
+	// If the user has vat dai directly (after performing vault actions, for instance), they don't need to create the DAI ERC20 just so we can burn it, we'll accept vat dai
+	function depositVatDai(uint256 attorontovatDai) external returns(uint256 attodaiHrd) {
```

Comment:
Switched this to take in `attorontovatDai` instead of attovatDai`.  The idea here is that this allows the user to sweep an account into DAI-HRD.  Also, if the user is calling this presumably they are already working with vatdai units anyway.

Reply:
I remember why i scaled it this way, and it was wrong. You end up depositing attoronto vatDai , but only end up getting attopot/attohrd. However, you are not rounded to 10**27, but to chi . even though it is a 10**27 number, it doesn't make sense to round to the 10**27

(this made sense when chi = ONE, but not in reality)